ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_np_arithmetic_operations.py

```diff
@@ -110,3 +110,42 @@ def test_numpy_subtract(
         subok=True,
         test_values=False,
     )
+    
+    
+#multiply
```

Comment:
can you please add the data generation part for hypothesis? you can follow similar to [example](https://github.com/unifyai/ivy/blob/fe1e0d78073ddad93162ece2cee4a8269985e5ca/ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_np_arithmetic_operations.py#L12)